weblate/static/loader-bootstrap.js

```diff
@@ -1380,6 +1380,33 @@ $(function () {
     });
   });
 
+  $("input[name='period']").daterangepicker(
+    {
+      autoApply: true,
+      startDate: moment(),
+      endDate: moment(),
+      alwaysShowCalendars: true,
+      opens: "left",
+      ranges: {
+        Today: [moment(), moment()],
+        Yesterday: [moment().subtract(1, "days"), moment().subtract(1, "days")],
+        "Last 7 Days": [moment().subtract(6, "days"), moment()],
+        "Last 30 Days": [moment().subtract(29, "days"), moment()],
+        "This Month": [moment().startOf("month"), moment().endOf("month")],
+        "Last Month": [
+          moment().subtract(1, "month").startOf("month"),
+          moment().subtract(1, "month").endOf("month"),
+        ],
+        "This Year": [moment().startOf("year"), moment().endOf("year")],
+        "Last Year": [
```

Comment:
Make the range labels translatable:

```suggestion
        gettext("Today"): [moment(), moment()],
        gettext("Yesterday"): [moment().subtract(1, "days"), moment().subtract(1, "days")],
        gettext("Last 7 days"): [moment().subtract(6, "days"), moment()],
        gettext("Last 30 days"): [moment().subtract(29, "days"), moment()],
        getttext("This month"): [moment().startOf("month"), moment().endOf("month")],
        gettext("Last month"): [
          moment().subtract(1, "month").startOf("month"),
          moment().subtract(1, "month").endOf("month"),
        ],
        gettext("This year"): [moment().startOf("year"), moment().endOf("year")],
        gettext("Last year"): [
```